src/main/java/org/isf/pricesothers/rest/PricesOthersController.java

```diff
@@ -0,0 +1,121 @@
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.isf.pricesothers.dto.PricesOthersDTO;
+import org.isf.pricesothers.manager.PricesOthersManager;
+import org.isf.pricesothers.mapper.PricesOthersMapper;
+import org.isf.pricesothers.model.PricesOthers;
+import org.isf.shared.exceptions.OHAPIException;
+import org.isf.utils.exception.OHServiceException;
+import org.isf.utils.exception.model.OHExceptionMessage;
+import org.isf.utils.exception.model.OHSeverityLevel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.swagger.annotations.Api;
+
+@RestController
+@Api(value = "/pricesothers", produces = MediaType.APPLICATION_JSON_VALUE)
+public class PricesOthersController {
+
+	@Autowired
+	protected PricesOthersManager pricesOthersManager;
+	
+	@Autowired
+	protected PricesOthersMapper mapper;
+
+	private final Logger logger = LoggerFactory.getLogger(PricesOthersController.class);
+
+	public PricesOthersController(PricesOthersManager pricesOthersManager, PricesOthersMapper pricesOthersmapper) {
+		this.pricesOthersManager = pricesOthersManager;
+		this.mapper = pricesOthersmapper;
+	}
+
+	/**
+	 * create a new {@link PricesOthers}
+	 * @param pricesOthersDTO
+	 * @return <code>true</code> if the prices others has been stored, <code>false</code> otherwise.
+	 * @throws OHServiceException
+	 */
+	@PostMapping(value = "/pricesothers", produces = MediaType.APPLICATION_JSON_VALUE)
+	ResponseEntity<String> newPricesOthers(@RequestBody PricesOthersDTO pricesOthersDTO) throws OHServiceException {
+		logger.info("Create prices others " + pricesOthersDTO.getCode());
+		boolean isCreated = pricesOthersManager.newOther(mapper.map2Model(pricesOthersDTO));
+		if (!isCreated) {
+			throw new OHAPIException(new OHExceptionMessage(null, "prices others is not created!", OHSeverityLevel.ERROR));
+		}
+		return ResponseEntity.status(HttpStatus.CREATED).body(pricesOthersDTO.getCode());
+	}
+
+	/**
+	 * Updates the specified {@link PricesOthers}.
+	 * @param pricesOthersDTO
+	 * @return <code>true</code> if the prices others has been updated, <code>false</code> otherwise.
+	 * @throws OHServiceException
+	 */
+	@PutMapping(value = "/pricesothers", produces = MediaType.APPLICATION_JSON_VALUE)
+	ResponseEntity<String> updatePricesOtherst(@RequestBody PricesOthersDTO pricesOthersDTO)
```

Comment:
Put Method should have code as @PathVariable. I know it should be an http standard.
`@PutMapping(value = "/pricesothers/{id}", produces = MediaType.APPLICATION_JSON_VALUE)`